pgxn/neon/libpagestore.c

```diff
@@ -407,8 +421,8 @@ pageserver_connect(shardno_t shard_no, int elevel)
 	{
 	case PS_Disconnected:
 	{
-		const char *keywords[5];
-		const char *values[5];
+		const char *keywords[16];
```

Comment:
Ctrl+F on the page finds 17 occurrences of `keywords[n_pgsql_params]`. So 17 should be the minimum size here